### PR TITLE
Add docs page for Create Next App

### PR DESCRIPTION
**Pull request by MehediH**
Following up from #14830, this PR adds a new page to the docs for Create Next App. The content is identical to the README created as part of #14830. 

Also added a link on the main `getting-started` page to the new docs for Create Next App to help users find more info on Create Next App if needed. 

I'm unsure as to whether the content for the documentation needs to be different from the one I wrote for the README. Please let me know if that's the case, and I will be happy to tweak the structure.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12.1s | 12.2s | ⚠️ +132ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.034 | 1.976 | -0.06 |
| / avg req/sec | 1229.35 | 1264.92 | +35.57 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.209 | 1.196 | -0.01 |
| /error-in-render avg req/sec | 2067.81 | 2090.05 | +22.24 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..d2f7.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-8da3811..3b61.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.6 kB | 87.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.1 kB | 6.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-43a8e8b..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 946 B | 946 B | ✓ |
| link.html gzip | 952 B | 952 B | ✓ |
| withRouter.html gzip | 938 B | 938 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| buildDuration | 13.5s | 13.2s | -255ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..d2f7.js gzip | 10.1 kB | 10.1 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-8da3811..3b61.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.6 kB | 87.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.1 kB | 6.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-43a8e8b..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 922 kB | 922 kB | ✓ |
| routerDirect.js | 916 kB | 916 kB | ✓ |
| withRouter.js | 916 kB | 916 kB | ✓ |
| Overall change | 4.52 MB | 4.52 MB | ✓ |

</details>

</details>
Commit: 36808d3e847b818017299717862a154d740f9257

File: docs/getting-started.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ yarn create next-app
 
 After the installation is complete, follow the instructions to start the development server. Try editing `pages/index.js` and see the result on your browser.
 
+For more information on how to use `create-next-app`, you can review the [tool's documentation here.](/docs/create-next-app.md)
```
Comment (timneutkens) [inline docs/getting-started.md:32]:
```suggestion
For more information on how to use `create-next-app`, you can review the [`create-next-app` documentation](/docs/create-next-app.md) 
```

File: docs/create-next-app.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+yarn create next-app
+```
+
+To create a new app in a specific folder, you can send a name as an argument. For example, the following command will create a new Next.js app called `blog-app` in a folder with the same name:
+
+```bash
+npx create-next-app blog-app
+# or
+yarn create next-app blog-app
+```
```
Comment (Timer) [inline docs/create-next-app.md:21]: I don't think we need this since we prompt you for it.

File: docs/create-next-app.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+### Options
+
+`create-next-app` comes with the following options:
+
+- **-e, --example [name]|[github-url]** - An example to bootstrap the app with. You can use an example name from the [Next.js repo](https://github.com/vercel/next.js/tree/master/examples) or a GitHub URL. The URL can use any branch and/or subdirectory.
+- **--example-path &lt;path-to-example&gt;** - In a rare case, your GitHub URL might contain a branch name with a slash (e.g. bug/fix-1) and the path to the example (e.g. foo/bar). In this case, you must specify the path to the example separately: `--example-path foo/bar`
```
Comment (Timer) [inline docs/create-next-app.md:28]: Inconsistent `<>` versus `[]` above.

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12.2s | -124ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.195 | 2.221 | ⚠️ +0.03 |
| / avg req/sec | 1138.8 | 1125.55 | ⚠️ -13.25 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.179 | 1.191 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2119.71 | 2098.35 | ⚠️ -21.36 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..adde.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-5932771..4350.js gzip | 6.73 kB | 6.73 kB | ✓ |
| polyfills-4c..dbb6.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.6 kB | 87.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.13 kB | 6.13 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-02cf3f6..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| polyfills-4c..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.7 kB | 82.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-ea5d131..c2be.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-4448c30..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 273 B | 273 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 554 B | 554 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 946 B | 946 B | ✓ |
| link.html gzip | 955 B | 955 B | ✓ |
| withRouter.html gzip | 940 B | 940 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| buildDuration | 13.6s | 13.8s | ⚠️ +178ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..adde.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-5932771..4350.js gzip | 6.73 kB | 6.73 kB | ✓ |
| polyfills-4c..dbb6.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.6 kB | 87.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.13 kB | 6.13 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-02cf3f6..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| polyfills-4c..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.7 kB | 82.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-ea5d131..c2be.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-4448c30..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 273 B | 273 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 554 B | 554 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: 173e9a8949f1db868afba5deb134fa4d07f4d90b

**Review by timneutkens — APPROVED** [review]

**Review by timneutkens — CHANGES_REQUESTED** [review]
Didn't see @Timer's review

File: docs/create-next-app.md (hunk 3 above)
Comment (timneutkens) [reply docs/create-next-app.md]: Fixed!

File: docs/create-next-app.md (hunk 2 above)
Comment (timneutkens) [reply docs/create-next-app.md]: Fixed!

**Review by timneutkens — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.6s | -138ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.272 | 2.255 | -0.02 |
| / avg req/sec | 1100.58 | 1108.75 | +8.17 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.265 | 1.214 | -0.05 |
| /error-in-render avg req/sec | 1976.79 | 2059.66 | +82.87 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..adde.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-5932771..4350.js gzip | 6.73 kB | 6.73 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.13 kB | 6.13 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-02cf3f6..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| polyfills-4c..dbb6.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-ea5d131..c2be.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-4448c30..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 273 B | 273 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 554 B | 554 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 946 B | 946 B | ✓ |
| link.html gzip | 955 B | 955 B | ✓ |
| withRouter.html gzip | 940 B | 940 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| buildDuration | 14s | 14.2s | ⚠️ +237ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..adde.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-5932771..4350.js gzip | 6.73 kB | 6.73 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.13 kB | 6.13 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-02cf3f6..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| polyfills-4c..dbb6.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-ea5d131..c2be.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-4448c30..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 273 B | 273 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 554 B | 554 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | MehediH/next.js canary | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: 61ce3269cec921def5540ad41ce59d0c9ff45919

**Comment by timneutkens** [comment]
Thanks @MehediH 💯

**Pull request merged**